easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3_config.patch

```diff
@@ -0,0 +1,17 @@
+diff -Nru augustus.orig/common.mk augustus/common.mk
```

Comment:
@Serpens Comment + author is still missing here?

Reply:
This patch is no longer needed after adding `buildopts`